release/_package.sh

```diff
@@ -33,20 +33,24 @@ function generate_release_properties_file {
 
 	local bundle_file_name="liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-tomcat-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}.7z"
 
-	local product_version="DXP ${_PRODUCT_VERSION^^}"
 
-	product_version="${product_version/-/ }"
 
 	local target_platform_version="${_PRODUCT_VERSION}"
 
 	if [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "dxp" ]
 	then
+		local product_version="DXP ${_PRODUCT_VERSION^^}"
```

Comment:
@Wykenz, declaring a variable as local inside an `if` and then declaring later, on line 52, won't be a variable redeclaration?

Why don't you do exactly as it's being done for the variable `target_platform_version`?

```
local product_version="${_PRODUCT_VERSION^^}"

if [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "dxp" ]
then
    product_version="DXP ${product_version}"
elif [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "portal" ]
    product_version="Portal ${product_version}"
fi
```